packages/ember-testing/lib/helpers.js

```diff
@@ -55,6 +55,7 @@ asyncHelper('visit', visit);
 
   @method click
   @param {String} selector jQuery selector for finding element on the DOM
+  @param {Object} context A DOM Element, Document, or jQuery to use as context
```

Comment:
@pixelhandler @jayphelps would using `Element | Document | jQuery` to cause issues with google closure compiler? (per @blesh #sobenlesh)